src/content/details/faqs.mdx

```diff
@@ -98,18 +97,18 @@ ___
 
 **Which instruction sets are supported and do we have to enable certain settings?**
 
-AVX2, AVX-512, and VNNI. The DeepSparse Engine will automatically utilize the most effective available
+AVX2, AVX-512, and VNNI. DeepSparse will automatically utilize the most effective available
```

Comment:
To confirm that this statement is true and makes sense @rsnm2 , it's applicable to ALL four components: Benchmark, Engine, Pipelines, and Server and not just the engine. Meaning, you should be able to substitute the phrase: "All four components will automatically utilize the most effective available instructions...." and this content makes sense.

Reply:
yeah this applies to all - benchmark, pipelines, and server use the same runtime